### ISSUE TITLE
Secret URLs for late submissions

### ISSUE THREAD

**Issue by paulmenzel** [indico/indico#6444]
Indico 3.3.2

I couldn’t find a way to generate a secret URL I can send to people wanting to submit an abstract, but the call for abstracts has been closed already.

**Comment by ThiefMaster** [comment]
Such a feature does not exist. Any reason against using the 'late submitters' functionality instead, where you can add the user ACL-style?

**Comment by ThiefMaster** [comment]
Invited abstracts are a good point. Maybe we should add a setting to let invited submitters (who are the only ones that not need an account to submit their abstract!) ignore the deadline in general...

**Comment by axu-trex** [comment]
Couldn't you just jump a few steps and manually create a Contribution for invited speakers? And then they can upload whatever into it themselves. They don't even have to have an account beforehand, just as long as the email address matches.

**Comment by paulmenzel** [comment]
Yes, there are always workarounds, but the more you do manually the more the chances are higher you do a mistake.
